hydra-node/src/Hydra/Chain/Direct/Tx.hs

```diff
@@ -457,69 +466,85 @@ abortTx networkId (Api.fromLedgerTxIn -> headInput, Api.fromLedgerData -> headDa
 
 -- * Observe Hydra Head transactions
 
+observeNetworkId :: Api.NetworkId
```

Comment:
Suggestion: Change the name, I don't think we are "observing" anything here, just hardcoding some value. Alternatively, pass the `NetworkId` to the functions using it?

Reply:
Resolved in https://github.com/input-output-hk/hydra-poc/pull/152/commits/1150b4a80ca614583bd1879a83752532e48b5f26